src/styles/block/cards.scss

```diff
@@ -0,0 +1,108 @@
+.main {
+  box-sizing: border-box;
```

Comment:
Move it to the `*` selector, if you don't know how to do it - put your question to the fechat_1